tvtk/plugins/scene/ui/actions.py

```diff
@@ -24,15 +28,14 @@ class SceneAction(Action):
 
     def _get_scene_manager(self):
         """ Trait property getter. """
-
+        #return self.task.scene_manager
         from tvtk.plugins.scene.i_scene_manager import (
             ISceneManager
         )
-
-        return self.window.get_service(ISceneManager)
+        return self.application.get_service(ISceneManager)
```

Comment:
I don't like this architecture / needing the application as it makes testing these actions more of a headache like Corran has mentioned, but previously these actions needed a WorkbenchWindow.  
The actions are set up to need either the application, the task or the task window (all of which are linked and can be obtained from each other).